src/juicebox_assembly_converter.py

```diff
@@ -0,0 +1,385 @@
+#!usr/bin/python
```

Comment:
This is more portable and explicit:
`#!/usr/bin/env python2`
See https://stackoverflow.com/questions/6908143/should-i-put-shebang-in-python-scripts-and-what-form-should-it-take